spec/unit/parse_yaml_spec.rb

```diff
@@ -7,10 +7,6 @@
       expect(process(nil).body).to be_nil
     end
 
-    it "returns false for empty body" do
-      expect(process('').body).to be false
-    end
-
```

Comment:
My understanding is that the new safe_yaml is supposed to return `nil`. Can we test that rather than deleting the test?